src/Umbraco.Core/Webhooks/WebhookEventType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Umbraco.Cms.Core.Webhooks;
+
+public enum WebhookEventType
+{
+    None,
+    Content,
+    Media,
+    Member,
+}
```

Comment:
If we are restricting WebHookEventType to an enum, what happens when the event type is something custom (or even another bit of umbraco) .eg Languages ? , DataTypes , etc ? 

Could it be the entityTypeName (which is a string) - so its still defined, for content etc, but it gives people the freedom to have a custom one ?

Reply:
This is purely used for frontend, so I'm not really sure why you'd need a custom one? 🤔 
This makes it so that we can filter the events in the `eventPicker` by event type.

Reply:
I guess in theory if you could replace the `eventPicker` logic on client side, then it would make sense, can you do that in angular ? 🤔

Reply:
Ah nvm I was overthinking it, i see now. 
So you could do your own `uSync` type, and when you selected one, it would only show the uSync relevant ones.....

Good point,  should this just be strings instead of enums perhaps?

Reply:
yeah not sure, but its a requirement of all IWebhookEvents ? 

If in the future the UI is expanded to do more (maybe custom stuff, maybe languages etc) then this would need refactoring, (meaning it would be breaking for people?) and if it was custom, / entityType name it wouldn't and it opens up options, people could for  example use this to do diffrent things with webhooks depending on what has happend in their notification (thinking custom here)

e.g when my custom notification fires, my handler might want to fire for webhooks of eventTypeA but not eventTypeB 
(but both webhook eventTypeA and eventTypeB might be fired for the same notification?)
and this not being restricted would let me do that? 
(just thinking out loud really - because this too could be done with two webhooks.).

Reply:
yes - that 😄